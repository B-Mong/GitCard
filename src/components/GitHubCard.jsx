import Card from 'react-bootstrap/Card'
import mike from './mike.jpg'

function GitHubCard() {
    return (
        <div>
            <Card style={{ width: "18rem" }}>
                <Card.Img variant="top" src={mike} />
                <Card.Body>
                    <Card.Title>Brian M</Card.Title>
                    <Card.Text>https://github.com/B-Mong</Card.Text>
                </Card.Body>
            </Card>
        </div>
    )
}

export default GitHubCard