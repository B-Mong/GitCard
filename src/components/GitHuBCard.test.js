import GitHubCard from './GitHubCard'
import renderer from 'react-test-renderer'

test("creates a snapshot", () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})