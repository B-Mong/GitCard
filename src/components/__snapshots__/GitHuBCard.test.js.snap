// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a snapshot 1`] = `
<div>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="mike.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Brian M
      </div>
      <p
        className="card-text"
      >
        https://github.com/B-Mong
      </p>
    </div>
  </div>
</div>
`;
